### ISSUE TITLE
Max entries age per feed

### ISSUE THREAD

**Issue by agios** [Athou/commafeed#1547]
**Is your feature request related to a problem? Please describe.**
There are some feeds that have a low volume of entries, but the old entries can still be relevant. Currently if cleanup is configured, it affects all feeds.

**Describe the solution you'd like**
If we could set the maxEntriesAgeDays per feed, we could have it longer or disabled for specific feeds, while others can be cleaned up more frequently.

**Comment by Athou** [comment]
This is not trivial because feeds are shared with all users that are subscribed to them (see [here](https://github.com/Athou/commafeed/issues/1466#issuecomment-2194558850) and [here](https://github.com/Athou/commafeed/issues/1509#issuecomment-2295771405)).

You could set `commafeed.database.cleanup.entries-max-age` to 0 though and let `commafeed.database.cleanup.max-feed-capacity` cleanup feeds that are over capacity.

**Comment by agios** [comment]
Thank you, that will work!

Can this also be overriden with an env variable which is easier for my docker deployment? In that case would it be `COMMAFEED_DATABASE_CLEANUP_ENTRIES_MAX_AGE`?

**Comment by Athou** [comment]
> Can this also be overriden with an env variable which is easier for my docker deployment? In that case would it be `COMMAFEED_DATABASE_CLEANUP_ENTRIES_MAX_AGE`?

Yep, see https://github.com/Athou/commafeed/blob/master/commafeed-server/doc/commafeed.adoc

**Comment by agios** [comment]
Thanks again!
